File: gameserver/messages/message.py
import messages.proto.gen.messages.proto.messages_pb2 as Proto
from entities.entity import *
class Message:
    
    @staticmethod
    def createEntitiesMessage(entities : Entity):
        protoMessage = Proto.Message()
        protoEntities = Proto.Entities()
        for e in entities:
            protoEntities.entities.append(e.toProto())

        protoMessage.entities.CopyFrom(protoEntities)
        return protoMessage

File: flareexample/lib/communication/message.dart
import 'package:flareexample/communication/proto/gen/lib/communication/proto/messages.pb.dart'
    as Proto;
import 'package:flareexample/entities/entity.dart';

class Message {
  static Proto.Message createEntitiesMessage(List<Entity> entities) {
    Proto.Entities protoEntitiesMessage = Proto.Entities();
    Proto.Message message = Proto.Message();
    entities.forEach((Entity entity) {
      protoEntitiesMessage.entities.add(entity.toProto());
    });
    message.entities = protoEntitiesMessage;
    return message;
  }
}
